src/hooks/chats/chatSidebar.ts

```diff
@@ -11,10 +11,11 @@ import {
   setChatSidebarVisibility,
 } from "store/actions/Chat";
 
-import { useSelector } from "./useSelector";
-import { useDispatch } from "./useDispatch";
-import { useNumberOfUnreadChats } from "./privateChats";
+import { useSelector } from "hooks/useSelector";
+import { useDispatch } from "hooks/useDispatch";
 import { AnyVenue } from "types/venues";
```

Comment:
types before hooks

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1922/commits/42d64d4216b50652324e863f758ed61c28ac6c17